app/src/editor/shared/components/modules/Scheduler.jsx

```diff
@@ -5,7 +5,7 @@ import { arrayMove } from 'react-sortable-hoc'
 import SortableList from '$shared/components/SortableList'
 import { withHover } from '$shared/components/WithHover'
 import SvgIcon from '$shared/components/SvgIcon'
-import TextControl from '$shared/components/TextControl'
+import { Text } from '$shared/components/Input'
```

Comment:
`<Text />` is `<input />`. No styling. No nothing. Just the unvisible enhancements like `onCommit`, `selectAllOnFocus`, etc. FYI.